README.md

```diff
@@ -127,6 +129,30 @@ Exceptions are not thrown unless calling gum itself does, the status code is int
 
 All of the rest of the options and usecases _should work_ ™. Please raise issues/PRs for any improvements. Much appreciated!
 
+## Simplified AP
```

Comment:
Lets call it `Higher Level` API than Simplified. I think that signals intention better?

Reply:
True